src/tokenizer.jl

```diff
@@ -1,9 +1,33 @@
-##############################################################################
-#
-# Split string into tokens on whitespace
```

Comment:
I went through the git blame, this dates 7 years ago - back then there was no offline documentation support in julia (>=julia 0.4). My guess is - these were the attempts to add comments explaining the code. I feel these won't be needed now in the codebase. 
What are your view on this, @aviks

Reply:
Yeah, I think so too. But some of these comments don't have proper docs, so we need to write them up instead of taking from official docs.